src/nrnoc/multicore.h

```diff
@@ -27,6 +27,10 @@ actual_v, etc.
 */
 
 #include "membfunc.h"
+namespace Eigen {
+template <typename _Scalar, int _Options, typename _StorageIndex>
+class SparseMatrix;
+}
```

Comment:
This file is installed and use, so if we include `<Eigen>` we need to install `Eigen` header too...